tests/components/conversation/test_init.py

```diff
@@ -664,16 +650,13 @@ async def test_http_api_wrong_data(
 
 async def test_custom_agent(
     hass: HomeAssistant,
+    init_components,
```

Comment:
Can we use `@pytest.mark.usefixtures("init_components")` instead so we don't have a unused variable